modules/dailymotionBidAdapter.js

```diff
@@ -42,11 +42,15 @@ function getVideoMetadata(bidRequest, bidderRequest) {
     iabcat2: Array.isArray(videoParams.iabcat2)
       ? videoParams.iabcat2
       : Object.keys(iabcat2),
-    id: videoParams.id || '',
-    lang: videoParams.lang || '',
+    id: videoParams.id ||
+      deepAccess(bidderRequest, 'ortb2.site.content.id', ''),
+    lang: videoParams.lang ||
+      deepAccess(bidderRequest, 'ortb2.site.content.language', ''),
     private: videoParams.private || false,
-    tags: videoParams.tags || '',
-    title: videoParams.title || '',
+    tags: videoParams.tags ||
+      deepAccess(bidderRequest, 'ortb2.site.content.keywords', []).join(','),
```

Comment:
Maybe more `.split(',')` instead of `.join(',')` ? Because `ortb2.site.content.keywords` seems contains a string and `tags` seems want an array.

Reply:
Oh you are right! Will change that